### DISCUSSION TITLE
Is there a deterministic order the documents will be in from LoadStartingWith?

### DISCUSSION THREAD

**Discussion by kamranayub** [ravendb/ravendb#12993] [Q&A] [answered]
The documentation doesn't mention this but what is the order of documents loaded using `session.Advanced.LoadStartingWith`? e.g. if I have documents with IDs ending in a sortable string, will results come back in ascending order? And if so, is there an easy way to reverse that (e.g. "Get 25 newest documents")? Or should we rely on a query/index (in my case I have other good reasons to use an index, but I was curious)?

**Comment by ppekrol** [comment, marked as answer]
Hi, this is returned by Id (ascending) as you wrote. There is no easy way to reverse that order at the moment. I think that index/query would be a better option to do it.

**Comment by ayende** [comment]
The results of `LoadStartingWith` is always in id order, ascending (UTF8 encoding). 
You can start from a particular location, but you cannot do an descending operation.
